### DISCUSSION TITLE
Removing a package from a changesets monorepo

### DISCUSSION THREAD

**Discussion by frolic** [changesets/changesets#1215] [Q&A]
We have a monorepo with many packages that have been using changesets for releases. We recently migrated one of the older packages to a new one and have deleted the old package.

However, changesets doesn't want to release because the now-deleted package is referenced in prior changesets MD files:

```
error Error: "red-turkeys-develop" changeset mentions a release for a package "@latticexyz/network" but such a package could not be found.
```

If I add the removed package to `ignore`, I get:

```
The package or glob expression "@latticexyz/network" is specified in the `ignore` option but it is not found in the project. You may have misspelled the package name or provided an invalid glob expression. Note that glob expressions must be defined according to https://www.npmjs.com/package/micromatch.
```

It sounds like a workaround might be adding the `package.json` back in with `private: true`, but we'd like to avoid cluttering the monorepo with empty packages. It also sounds like we could remove references to `@latticexyz/network` in all changesets, but that feels wrong too, since it would muck with our changelog/history of changes.

How should I remove a package from our changesets pipeline so we aren't continuing to publish it?

**Comment by enesozturk** [comment]
+1 @Andarist having the exact same issue as well, couldn't find any relevant solutions on the repo. Do you have any workaround? 🙏🏽
